docs/standard-library/allocators.md

```diff
@@ -18,6 +18,35 @@ class vector
 
 The C++ Standard Library provides a default implementation for an allocator. In C++11 and later, the default allocator is updated to expose a smaller interface; the new allocator is called a *minimal allocator*. In particular, the minimal allocator's `construct()` member supports move semantics, which can greatly improve performance. In most cases, this default allocator should be sufficient. In C++11 all the Standard Library types and functions that take an allocator type parameter support the minimal allocator interface, including `std::function`, `shared_ptr, allocate_shared()`, and `basic_string`.  For more information on the default allocator, see [`allocator` Class](allocator-class.md).
 
+## Writing Your Own Allocator (C++20)
```

Comment:
This added example is depicting usage of `std::allocator`; it is not depicting how a user would write their own allocator in C++20 (which is very similar to how it would be written in C++11, except for things like `operator!=` being unnecessary to provide).